mernarezk/assignment4/CountIslandsTest.java

```diff
@@ -0,0 +1,23 @@
+package Assignment4;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+class CountIslandsTest {
+
+	@Test
+	void test() {
+		CountIslands tester = new CountIslands();
+		boolean grid[][] = new boolean[][] { { false, true, false, true }, { true, true, false, false },

```

Comment:
Maybe fix the formatting so it's easier to see what's going on in the grid? Like every 'row' in a new line, so it's easy to see where the islands are.

Reply:
Done